extensions/sql-bindings/src/common/constants.ts

```diff
@@ -74,10 +74,16 @@ export const openFile = localize('openFile', "Open File");
 export const closeButton = localize('closeButton', "Close");
 export const enterPasswordPrompt = localize('enterPasswordPrompt', '(Optional) Enter connection password to save in local.settings.json');
 export const connectionProgressTitle = localize('connectionProgressTitle', "Testing SQL Server connection...");
-export const enterObjectName = localize('enterObjectName', 'Enter SQL table or view to query');
-export const enterObjectNameToUpsert = localize('enterObjectNameToUpsert', 'Enter SQL table to upsert into');
+export const enterSqlObjectName = localize('enterSqlObjectName', 'Enter SQL table to query');
+export const enterViewsObjectName = localize('enterViewsObjectName', 'Enter SQL view to query');
+export const enterSqlObjectNameToUpsert = localize('enterObjectNameToUpsert', 'Enter SQL table to upsert into');
 export const selectTable = localize('selectTable', 'Select table to use');
+export const selectView = localize('selectView', 'Select view to use');
 export const tableListProgressTitle = localize('tableListProgressTitle', "Fetching tables for selected database...");
+export const viewListProgressTitle = localize('viewListProgressTitle', "Fetching views for selected database...");
+export const sqlTable = localize('sqlTable', 'Table');
```

Comment:
Just name these `view` and `table` - the names should generally match what the actual string is going to be